frontend/src/features/idir/search/components/IDIRCompanySearchResults.tsx

```diff
@@ -153,8 +155,38 @@ export const IDIRCompanySearchResults = memo(
     });
 
     return (
-      <div className="table-container idir-search-results">
-        <MaterialReactTable table={table} />
+      <div className="table-container idir-company-search-results">
+        {data?.items?.length !== 0 && <MaterialReactTable table={table} />}
+        {data?.items?.length === 0 &&
+          <>
+            <Stack style={{ width: "900px" }} spacing={2}>
+              <NoRecordsFound />
+              <Box
+                display="flex"
+                justifyContent="center"
+                alignItems="center"
+                minHeight="30vh"
```

Comment:
Try not to use min-height, and see if paddings are needed (depending on the wireframe specs)